lib/containers/podman.pm

```diff
@@ -31,8 +31,7 @@ sub configure_insecure_registries {
 }
 
 sub get_storage_driver {
-    my $json = shift->info(json => 1);
-    my $storage = $json->{store}->{graphDriverName};
+    my $storage = script_output("podman info -f '{{.Store.GraphDriverName}}'");
```

Comment:
What happens if script_output result is null? In the original code the test would die, right?

Reply:
I grep'ed for all calls to this functions and all check against expected values:

- https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/containers/validate_btrfs.pm#L48
- https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/containers/rootless_podman.pm#L105